src/utils/api.js

```diff
@@ -1,7 +1,22 @@
 /* @flow */
+import { getToken } from './tokenStorage';
+
+export type LoginType = {
+  email: string,
+  password: string,
+};
+
+export type PromiseFormType = {
+  name: string,
+  shortDescription: string,
+  longDescription: string,
+  parliamentId: number,
+};
 
 export const checkStatus = (response: Response) => {
-  if (response.status >= 200 && response.status < 300) { //eslint-disable-line
+  /* eslint-disable no-magic-numbers */
+  if (response.status >= 200 && response.status < 300) {
```

Comment:
Is there some sort of javascript library that allow us to reference different status codes without having magic numbers? Django and Dropwizard (java) have something like "Response.Status.OK" for 200 for example. Might make things more readable for the future? (Not suggesting a change but might be nice?)